show/chassis_modules.py

```diff
@@ -59,13 +60,14 @@ def status(db, chassis_module_name):
         desc = data_dict[CHASSIS_MODULE_INFO_DESC_FIELD]
         slot = data_dict[CHASSIS_MODULE_INFO_SLOT_FIELD]
         oper_status = data_dict[CHASSIS_MODULE_INFO_OPERSTATUS_FIELD]
+        serial = data_dict[CHASSIS_MODULE_INFO_SERIAL_FIELD]
```

Comment:
Is there any chance of this submodule being updated in a workspace without sonic-platform-daemons being updated? If so, there will be no `CHASSIS_MODULE_INFO_SERIAL_FIELD` key in the data and this will crash. If there's no risk of this mismatch of submodules, then it looks good to me.

Reply:
Yeah, both submodules are pulled on make command. I am not sure how these are updated, let me check on that

Reply:
So, we don't handle sub-modules update, we use the external ref for pulling ```sonic-buildimage```, and as long as it is synced in upstream ```sonic-buildimage``` to use the correct and updated refs for these sub-modules, we should be okay

For upstream PRs, I will add the dependency links and note that it should go together

Reply:
Right, it needs to be synced upstream. As long as that happens, this code is fine. If that can't be guaranteed, simply changing to `data_dict.get(CHASSIS_MODULE_INFO_SERIAL_FIELD, 'N/A')` should be enough to make sure there's no crash. I'd keep this as-is, and only change upon guidance from MSFT; ask about this when they review.

Reply:
Will do